pint/models/parameter.py

```diff
@@ -134,6 +138,17 @@ def value(self,val):
                     raise ValueError("The ._num_value has to be a pure number or None. "\
                                      "Please check your .get_num_value method. ")
 
+    def prior_probability(self,value=None):
+        """Return the prior probability, evaluated at the current value of
+        the parameter, or at a proposed value.
+        
+        Probabilities are evaluated using the num_value attribute
+        """
+        if value is None:
+            return self.prior.prior_probability(self.num_value)
+        else:
+            return self.prior.prior_probability(value)
+            
```

Comment:
I used to not like it, but it has grown on me.  And unlike passive voice, it is almost always shorter.  And in this particular case, I like how there is only 1 return call.